packages/ui-components/src/components/UIFlexibleTable/UIFlexibleTable.scss

```diff
@@ -318,9 +324,24 @@ $readonly-labels-opacity: 0.4;
                 }
             }
         }
+
+        &-table.dragged {
+            .flexible-table-content-table-row.highlight-drop-warning {
+                .flexible-table-content-table-row-wrapper::before {
+                    position: absolute;
+                    width: calc(100% - 2px);
+                    height: calc(100% - 2px);
```

Comment:
I think we can avoid 'calc'
I think it would be cleaner with following and it would automatically adjust to border size increasement:
```
width: 100%;
height: 100%;
box-sizing: border-box;
border: 1px solid $warning-highlight-color;
```

Reply:
Update is done